routes/contactSave.js

```diff
@@ -673,6 +654,12 @@ function post(req, res, next) {
             }
             if (setVerified) {
               profile.verified = newVerified;
+            
+            }
+            if(setVerifiedFlag){
```

Comment:
What do you think, Now it only changes if the setVerifiedFlag is true, thats only true when the user is verified by admin etc